cmd/observe/observe.go

```diff
@@ -77,11 +77,16 @@ var verdicts = []string{
 	pb.Verdict_ERROR.String(),
 }
 
-func eventTypes() (l []string) {
-	for t := range monitorAPI.MessageTypeNames {
-		l = append(l, t)
-	}
-	return
+// eventRypes are the valid event types supported by observe. This corresponds
+// to monitorAPI.MessageTypeNames, excluding MessageTypeNameAgent,
+// MessageTypeNameDebug and MessageTypeNameRecCapture. These message types have
+// separate sub-commands.
```

Comment:
nit: typo and rephrasing:

```suggestion
// eventTypes are the valid event types supported by observe. This corresponds
// to monitorAPI.MessageTypeNames excluding MessageTypeNameAgent,
// MessageTypeNameDebug, and MessageTypeNameRecCapture. Theses excluded
// message types are not supported by observe but rather have separate sub-commands.
```

Reply:
Thanks, I've slightly adjusted the suggestion and applied it.